src/Makefile

```diff
@@ -270,6 +268,11 @@ endif
 	--zone $(GCLOUD_ZONE) \
 	--project $(GOOGLE_PROJECT_ID)
 
+rollback-helm:
+	# Rollback to the previous good release
```

Comment:
Helm will look specifically for the staging images, because it uses [`HELM_RELEASE`](https://github.com/greenpeace/planet4-builder/blob/develop/src/helm_rollback.sh#L7) from the [env variables](https://github.com/greenpeace/planet4-nro-generator/blob/master/templates/nro/.circleci/config.yml.tmpl#L37).